RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Enable Ruff annotation check ANN206.
```

Comment:
```suggestion
This release adds missing annotations in class methods, improving
our type coverage.
```

Let's also update this to make it easier for our users to understand what changed 😊

Reply:
Sorted, thank you!